frontend/packages/dev-console/src/components/NamespacedPage.scss

```diff
@@ -15,6 +15,11 @@
     }
   }
 
+  &__toolbar-options {
+    flex-grow: 1;
+    text-align: right;
+  }
```

Comment:
If you see my other comment - this is no longer needed. I just want to point out a problem here for future knowledge sake.

`flex-grow` here in order to allow you to place the content on the far right will have unintended side-effects. Flex grow is a weight based system... if two items have it set to the same number, they'll equally share the available space and you'll end up with a lot of whitespace being unused by your `toolbar-options` (since they are smaller items). 

`text-align` shouldn't be needed too often when it comes to flex layouts, simply because they have the power to align baked in. 

Just remember the uses for the main flex properties:

- `flex-grow` is for when you want your item to expand to take up more space if it's available (or to balance against another `flex-grow` sibling)
- `flex-shrink` is for when you are okay taking less space if it's needed (you could think of this as "I'm okay to share my space if someone else needs it")
- `flex-basis` is for the heart of the initial space. When rendering something into a `flex-basis` of `auto`, you're saying "before any other calculations, try to give me the space I need by default" so icons and other fixed-sized items are given their space before `flex-shrink` or `flex-grow` is calculated.

I have found reading through the [CSS-Tricks page on Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) to be very helpful.